documents/policies/external/Competition Requirements.md

```diff
@@ -15,25 +15,30 @@ An official WCA Competition must:
 3. Comply with all [WCA Policies](wca{documents}).
 4. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue. 
    1. Any restrictions placed on competitors and/or spectators who can attend must be publicly outlined in the competition information section and approved by the WCA Board or WCA Competition Announcement Team.
+   2. Individuals suspended from attending WCA sanctioned competitions may not attend any WCA sanctioned competitions until the period of their suspension has expired (see 1.1 of Suspensions and other Sanctions).
+   3. A WCA Delegate may require individuals to leave the competition venue for behavior that is unlawful/violent/indecent/unsafe, or intentionally damages venue facilities or personal property within the venue. Delegates must take into account the seriousness of the situation and the best possible course of action. Delegates must also report any such incidents to the WDC to investigate the need for disciplinary action.
```

Comment:
Hi Ed, this was added because of WDC feedback.

Reply:
I assumed so - but I don't want to create the possibility of WDC not following the policy. I feel like the word "investigate" is a strong word - the WDC may not decide it isappropriate to open an "investigation" (e.g. because of the nature of the behaviour, or the actions already taken at the comp), or an investigation may already in progress and this is additional information to it.

Similarly, Delegates should be aware of local laws and venue restrictions. If the competition is in certain public places and depending on the local laws it may not be possible for the Delegate to unilaterally require an individual to leave. 

Additionally, now that I think about it, I'm uncomfortable with the idea that the WCA Delegate (and individual acting in their capacity as a representative of the WCA) decides on this, as this opens up the WCA to legal action for events it doesn't organise. Wouldn't it be better for the competition organiser to be the ones who decide on this?

Reply:
Much of this is outside of the WDC scope, so I will not respond to those portions. 

The WDC will need to look into any of these reports and whether or not further sanctions should apply, so I believe the language is appropriate. 

I am not in favor of all organizers having the sole power to remove people from venues.

Reply:
Sorry for the double post, but I am fine changing the word to “determine” over investigate if it really makes an impact.

Also, 11h already allows delegates to ask people to leave. This is just creating consistency between documents.

Reply:
I think the wording there is up to WDC - but my suggestion with policies is don't force yourself to do something of you can envisage a circumstance where it wouldn't be appropriate.

On the language: in general the WCA has far too much emphasis on the WCA Delegate being the one to decide anything of importance. Things are structured with the idea that the organiser is a 13 year old child who is being particularly helpful at that comp but all the serious stuff is done by the Delegate - so serious tasks are therefore the domain of the WCA. But the WCA isn't here to organise competitions - we just make sure they follow a set of standards, host the results, and provide software tools to facilitate their organisation. 

The real organisers are the key Delegates, or entities like Speedcubing Australia, Cube New England, etc. They should absolutely have the ability to run their events and handle issues not related to the cubing competition aspect, while liaising with WCA representatives to ensure WCA policies are followed. It's going to take a lot of effort to update WCAs thinking and documents for this, but I at least don't want more wording to be added that makes it harder. This isn't to say that particularly helpful 13 year old cubers who are a key volunteer for a given competition don't have a role to play, but let's not lie to ourselves and say they're the organiser - most WCA Delegates are organisers. We need to do a better job of separating the two "hats" to empower whichever local cubing organisation is coordinating things, and better define the WCAs role as a global governing body. The WCA having completed consistency and in essence organising competitions is simply not scalable.

Reply:
I have let Zeke know I don’t really have an opinion either way. The WDC will review each report regardless of the language and determine if the action the delegates took was sufficient or if further sanctions are needed.

I don’t agree or disagree with your other comments but documents should be consistent. The regulations already contain effectively the same language. This is a larger conversation that cannot possibly be addressed by a single point in the WCRP.